frontend/src/index.css

```
@@ -1603,19 +1726,18 @@ textarea {
-  width: 3rem;
-  height: 3rem;
+  width: 1.25rem;
+  height: 1.25rem;
   flex-shrink: 0;
   align-items: center;
   justify-content: center;
-  border-radius: var(--aurral-radius-round);
-  background: var(--aurral-surface);
-  color: var(--aurral-white);
+  background: transparent;
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)